cmd/posix.go

```diff
@@ -34,9 +34,9 @@ import (
 )
 
 const (
-	fsMinFreeSpace         = 1024 * 1024 * 1024
-	fsMinFreeInodesPercent = 5
-	maxAllowedIOError      = 5
+	fsMinFreeSpace    = 1024 * 1024 * 1024 // Min 1GiB free space.
+	fsMinFreeInodes   = 10000 + 2          // Min 10000 (max multiparts) + 2 (uploads.json, xl.json or fs.json) free inodes.
```

Comment:
you could keep it as `fsMinFreeInodes   = 10000` removing `+ 2` the extra `2` isn't much adding value